contracts/router/src/contract.rs

```diff
@@ -234,7 +234,9 @@ fn assert_minimum_receive(
             amount: swap_amount,
         })
     } else {
-        Ok(Response::default())
+        Ok(Response::default().set_data(to_binary(&SwapResponseData {
+            return_amount: swap_amount,
+        })?))
```

Comment:
Not a good idea to set it only if `minimum_receive` is set. In general, the response should be set in all successful executions.

Reply:
Certainly. Makes sense